docs/src/Sticky.doc.js

```diff
@@ -42,9 +42,9 @@ card(
         description: `Use numbers for pixels: top={100} and strings for percentages: top="100%"`,
       },
       {
-        name: 'dangerouslySetZIndex',
-        type: '{ __zIndex: number }',
-        defaultValue: '{ __zIndex: 1 }',
+        name: 'zIndex',
+        type: 'interface { index(): number; }',
```

Comment:
Can we write this type another way like `FixedZIndex | CompositeZIndex` or `Indexable` to make it more obvious how this should be used when scanning the docs?